source/_patterns/molecules/blocks/hero/hero.twig

```diff
@@ -1,5 +1,12 @@
+{% set url_attributes = url_attributes ? url_attributes : create_attribute() %}
+
 <div class="jcc-hero jcc-hero--{{ background_variant }}">
   <div class="jcc-hero__container">
+    {% include '@molecules/navigation/breadcrumbs/breadcrumbs.twig' with {
+        url: breadcrumbs.url,
+        name: breadcrumbs.name
+    } %}
+
```

Comment:
This one needs to be an array, so we can pass both of these links:

<img width="439" alt="Screen Shot 2019-10-01 at 3 55 03 PM" src="https://user-images.githubusercontent.com/60979/65996085-bcfbec80-e464-11e9-90e6-fa3e855fb273.png">

I would just make the same breadcrumb data structure you have available to the hero component, so ultimately the result would be:

```twig
{% include '@molecules/navigation/breadcrumbs/breadcrumbs.twig' with { breadcrumbs: breadcrumbs } %}
```

or if you want to be less verbose:

```twig
{% include '@molecules/navigation/breadcrumbs/breadcrumbs.twig' %}
```